R/AllGenerics.R

```diff
@@ -35,7 +35,7 @@ function (x, value) standardGeneric("startDate<-"), signature="x")
 setGeneric("endDate", function (x) standardGeneric("endDate"))
 setGeneric("endDate<-",
     function (x, value) standardGeneric("endDate<-"), signature="x")
-setGeneric("alias")
+setGeneric("alias", function (object) standardGeneric("alias"))
```

Comment:
The other attribute getters are `function (x)` so I'd follow that for consistency.

Reply:
I changed, it, but then saw [the documentation](https://crunch.io/r/crunch/reference/describe.html) that said "Same as `x` but for the alias method, in order to match the generic from another package." and didn't want to wantonly over-rule that.

I don't think there's that much benefit in keeping `crunch::alias` in line with `stats::alias`, but that was the original reason. I'll harmonize this with the rest unless there's an objection and we should maintain it.

Reply:
Oh I see. The benefit of keeping it in line with `stats::alias` is so that we aren't masking that function (whatever it is). So let's keep it as is. I guess I'm wondering now though, why did the generic have to change at all here?

Reply:
It looks like it didn't, for some reason I was having difficulties with it earlier passing `CMD check` but it does now without any editing, so we're good.